README.md

```diff
@@ -176,3 +138,66 @@ outgoing PDUs, and is used to construct the topic
+
+## Identifiers
+
+This is a listing of all identifiers (topics, ids, etc.) involved when using the example brash_docker (or multihost) configuration
+- See the Comands section below for sample commands corresponding to each cmd column with an entry.  Blank cells represent combinations that are not applicable, or not currently possible.
+- The Cfe CH column is used internally by the CFE CF application and is not currently used for ros-to-ros transfers.
+- 'gsw-topic' refers to the topic generated on rosgsw, with 'cmd' being messages sent out and 'tlm' being packets received
+- 'sbn'-topic' similarly refers to rosfsw topics as sent on the SBN bridge ('tlm') or received ('cmd'). Note: Despite the naming convention, the CCSDS packets can be of CMD or TLM variants in either direction.
+- This table reflects the IDs/topics for messages flowing from src to dst only. Any replies, including ACKs/NAKs, will use the MIDS/topics for the row corresponding to that message flow.
+
+| Src    | Dst    | Cfe Ch | MID  | gsw-topic-cmd | gsw-topic-tlm | sbn-topic-tlm | sbn-topic-cmd | Note                     | rosgsw cmd | rosfsw cmd | cfe cmd via rosgsw |
+|--------|--------|--------|------|---------------|---------------|---------------|---------------|--------------------------|------------|------------|--------------------|
+| rosgsw | rosfsw | 2      | 18ca | flightsystem  |               | flightsystem  |               |                          | put        | get        |                    |
+| rosgsw | fsw    | 0      | 18c8 | cfe           |               |               |               |                          | put        |            |                    |
+| fsw    | rosfsw | 1      | 8c3  |               |               | flightsystem  |               |                          |            |            | TX                 |
+| fsw    | rosgsw | 0      | 8c2  |               | groundsystem  |               |               |                          |            |            | TX                 |
+| rosfsw | rosgsw | 2      | 8c4  |               | groundsystem  |               | groundsystem  |                          | get        | put        |                    |
+| rosfsw | fsw    | 1      | 18c9 |               |               |               | cfe           | UNACK Only (Known Issue) |            | put        |                    |
+
+
+## Commands
+
+The following series of example commands starts 2 instances of the service under a single ROS machine and executes a series of commands.
+
+Start CFDP ROS Fsw and Gsw instances using explicit parameters.  Default values are suitable for RosFsw instance when RosGsw is on a discrete Ros machine.
+- `ros2 run cfdp_wrapper cfdp_wrapper.py --ros-args -r __node:=cfdpgsw -p entityID:=1 -p "filestore:=cfdp/rosgsw"`
+- `ros2 run cfdp_wrapper cfdp_wrapper.py --ros-args -r __node:=cfdpfsw -p entityID:=2 -p altServices:=True -p "filestore:=cfdp/rosfsw" -p "pduTopicPrefix:=/rosfsw/cfdp/pdu/entity"`
```

Comment:
Could you describe what altServices:=True and the -p options stand for?

Reply:
These are alternate directions for starting CFDP manually when not using the Docker configuration, and may need to be updated.  All parameters have descriptions defined with their declarations in cfdp_wrapper.py -- I'm not sure what ROS/documentation mechanism we have to access those.


entityID is the CFDP ID this instance should listen to. 

filestore is "The base path (relative to script launch location) for all local file operations"

altServices and pduTopicPrefix look to be deprecated - I think either Ana or Steve previously removed them. They were originally used to specify an alternate topic prefix to listen on when running multiple CFDP instances on a single machine to avoid conflicts. I believe the namespace prefixes are now automatically applied per ROS settings to take their place, though i have not tested that.